android/assets/jsons/translations/Simplified_Chinese.properties

```diff
@@ -129,9 +128,9 @@ Peace with [civName]? = 与[civName]议和吗？
 Very well. = 很好。
 Farewell. = 再见。
 Sounds good! = 听起来还不错。(同意)
-Not this time. = 还不是时候，我想这次不行。(拒绝)
-Excellent! = 好，成交！
-How about something else... = 哦！我想我们应该再来点儿别的......
+Not this time. = 这次不行，我只和有诚意的人交易。(拒绝)
+Excellent! = 真是荣幸之至！互相尊重、平等互利是建立友好关系的第一步。
+How about something else... = 谈判桌上的尔虞我诈才是我的兴趣所在。（进行协商）
```

Comment:
@xk730 Sorry for rolling back most of your changes here. The original civ5 translation is better.